src/js/views/patients/worklist/action_views.js

```diff
@@ -48,8 +48,8 @@ const ActionItemView = View.extend({
     this.flow = this.model.getFlow();
 
     this.listenTo(state, {
-      'select:multiple': this.render,
-      'select:none': this.render,
+      'select:multiple': this.showCheck,
+      'select:none': this.showCheck,
```

Comment:
After the bulk edit save is finished, the row items in the UI aren't updated.

To reproduce: Go to the worklist => select one or more list items => make a change (e.g. switch state from `To do` to `Doing` => hit `Save` button. When the bulk edit is finished, those list items aren't updated in the UI. And the list needs to be reloaded to see those changes (via a hard refresh of the app or leaving the worklist page and coming back).

Think it's related to this: https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/apps/patients/worklist/worklist_app.js#L166.

Before this PR, `state.clearSelected()` was called when the bulk edit is finished, which triggers `select:none` and a `this.render()` on the list item.

Reply:
Looks like the date/time are updated in the UI correctly. But the state and owner are not...? Now I'm confused...

Reply:
Good catch on this one.  I'm a little surprised that no tests failed with this.  I'm assuming what I just pushed will need line coverage, but either way I plan on adding tests to cover what should have failed here.

Reply:
From your last two commits, everything looks to be working well! Also was surprised the tests didn't catch it.

Reply:
There's a bit of event loop because some of the custom saves don't check for change..  so it works kind of.. that's why there tests are currently failing. :-/